app/api/dao/user.py

```diff
@@ -188,3 +190,79 @@ def authenticate(username_or_email, password):
             return user
 
         return None
+
+    @staticmethod
+    def get_achievements(user_id):
+        """
+        Gets all the completed tasks of the user and
+        return them in a list. Achievements are completed tasks
+
+        Args:
+            user_id: The ID of the user for whom tasks are
+                requested.
+
+        Returns:
+            achievements: A list containing the user's achievements
+        """
+        user = UserModel.find_by_id(user_id)
+        all_relations = user.mentor_relations + user.mentee_relations
+        tasks = []
+        for relation in all_relations:
+            tasks += relation.tasks_list.tasks
+        achievements = [task for task in tasks if task.get("is_done")]
+        return achievements
+
+    @staticmethod
+    def get_user_statistics(user_id):
+        """
```

Comment:
In front of `"""` you can have a small summary of the function

You can add this:
```
"""Shows some basic user statistics
```